.dev_scripts/benchmark/train_benchmark.sh

```diff
@@ -34,24 +34,24 @@ echo ${CONFIG} &
 ./tools/slurm_train.sh ${PARTITION} ${WORK_DIR} ${CONFIG} ${ROOT_DIR}/${WORK_DIR} 8 --cfg-options default_hooks.checkpoint.max_keep_ckpts=1 >/dev/null &
 
 # VIS
-CONFIG=configs/vis/masktrack_rcnn/masktrack_rcnn_r50_fpn_12e_youtubevis2019.py
-WORK_DIR=masktrack_rcnn_r50_fpn_12e_youtubevis2019
+CONFIG=configs/vis/masktrack_rcnn/masktrack-rcnn_resnet50-fpn_8x1bs-12e_youtubevis2019.py
```

Comment:
```suggestion
CONFIG=configs/vis/masktrack_rcnn/masktrack-rcnn_mask-rcnn-resnet50-fpn_8x1bs-12e_youtubevis2019.py
```
modify it for all masktrack rcnn configs